Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -127,7 +127,7 @@ public static class TestObjects {
 
     public class Car
     {
-        private string color;
+        public string color;
```

Comment:
Should this be internal instead of public?

Reply:
internalized.